src/Mod/TechDraw/Gui/Command.cpp

```diff
@@ -94,8 +94,8 @@ CmdTechDrawNewPageDef::CmdTechDrawNewPageDef()
 {
     sAppModule      = "TechDraw";
     sGroup          = QT_TR_NOOP("TechDraw");
-    sMenuText       = QT_TR_NOOP("Insert new default Page");
-    sToolTipText    = QT_TR_NOOP("Insert new default Page");
+    sMenuText       = QT_TR_NOOP("Insert new default page");
+	sToolTipText    = sMenuText;

```

Comment:
It seems you accidentally added a Tab as indentation. I would keep the style of the previous code and use only spaces.

Reply:
Thanks. this is now fixed.